plexpy/notification_handler.py

```diff
@@ -293,24 +310,26 @@ def build_notify_text(session, state):
         full_title = session['title']
 
     # Generate a combined transcode decision value
-    if session['video_decision']:
+    transcode_decision = ''
+    if session.get('video_decision', None):
```

Comment:
I don't think `.get` works with sqlite objects.

```
Traceback (most recent call last):
  File "/Users/tim/PyCharmProjects/plexpy-jw/plexpy/logger.py", line 210, in new_run
    old_run(*args, **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/Users/tim/PyCharmProjects/plexpy-jw/plexpy/notification_handler.py", line 128, in notify
    notify_strings = build_notify_text(session=stream_data, state=notify_action)
  File "/Users/tim/PyCharmProjects/plexpy-jw/plexpy/notification_handler.py", line 314, in build_notify_text
    if session.get('video_decision', None):
AttributeError: 'sqlite3.Row' object has no attribute 'get'
```

Any reason for this change?